OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -66,15 +66,33 @@ protected LabelWidget(LabelWidget other)
 			GetContrastColorLight = other.GetContrastColorLight;
 		}
 
-		public override void Draw()
+		// Allow same checks when ever we get thee font, should this be cached?
```

Comment:
Do we need to address anything here or has this already been figured out? If there is a todo the comment should be prefixed with a `TODO: `.

Reply:
These were actually left over from the original PR that I inherited to implement this. I'm unsure of their purpose, they can probably be removed tbh unless someone who understands what's going on better understands what they are referencing.